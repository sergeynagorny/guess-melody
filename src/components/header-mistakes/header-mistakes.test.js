import React from "react";
import renderer from "react-test-renderer";
import HeaderMistakes from "./header-mistakes.jsx";


describe(`Should HeaderMistakes render correctly`, () => {
  it(`With one zero count`, () => {
    const tree = renderer.create(
        <HeaderMistakes
          count={0}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`With one one count`, () => {
    const tree = renderer.create(
        <HeaderMistakes
          count={1}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
