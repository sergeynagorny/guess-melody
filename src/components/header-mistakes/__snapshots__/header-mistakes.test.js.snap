// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should HeaderMistakes render correctly With one one count 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;

exports[`Should HeaderMistakes render correctly With one zero count 1`] = `
<div
  className="game__mistakes"
/>
`;
